assembler.py

```diff
@@ -7,9 +7,109 @@
 parallel_mode = lambda x: x._parallel_mode
 
 
-def get_jobs(module):
+def get_tasks(module):
     tasks = sorted(
         [obj for obj in inspect.getmembers(module) if is_job(obj[1])],
         key=lambda x: parallel_mode(x[1])
     )
     return OrderedDict(tasks)
+
+
+class Assembly(object):
+    get_tasks = get_tasks
```

Comment:
This does the actual work of "dis-assmebling" (in a loose sense of the word) the module that is submitted by the user for parallel processing.